src/main/java/com/microsoft/azure/functions/worker/binding/RpcHttpRequestDataSource.java

```diff
@@ -86,4 +91,16 @@ public Builder createResponseBuilder(HttpStatus status) {
 			return new HttpRequestMessageImpl(v, bodyData.getValue());
 		});
 	}
+
+	private static Map<String, String> convertFromNullableMap(Map<String, NullableTypes.NullableString> nullableMap) {
+		if (Util.isTrue(System.getenv("FUNCTIONS_WORKER_NULLABLE_VALUES_ENABLED"))) {
```

Comment:
Let us make `FUNCTIONS_WORKER_NULLABLE_VALUES_ENABLED` a constant.

Reply:
nice catch! there is a constant file you can define it there.